src/sinks/aws_kinesis_streams/request_builder.rs

```diff
@@ -0,0 +1,108 @@
+use crate::buffers::Ackable;
+use crate::event::Event;
+use crate::event::{EventFinalizers, Finalizable};
+use crate::sinks::aws_kinesis_streams::sink::KinesisProcessedEvent;
+use crate::sinks::util::encoding::{EncodingConfig, StandardEncodings};
+use crate::sinks::util::{Compression, RequestBuilder};
+use bytes::Bytes;
+use rusoto_kinesis::PutRecordsRequestEntry;
+use std::io;
+use vector_core::ByteSizeOf;
+
+pub struct KinesisRequestBuilder {
```

Comment:
Are these structs used outside of this component? If not, I suggest `pub(super)`. Ditto for other structs, I see.

Reply:
What's the benefit of that? `src/sinks/aws_kinesis_streams/mod.rs` imports it privately, so it won't be accessible outside of this sink code anyway.

Reply:
Ah, gotcha, I missed that. No real benefit then, other than making it redundantly obvious.